src/app/child-dev-project/aser/model/aser.ts

```diff
@@ -22,21 +22,8 @@ import { DatabaseEntity } from '../../../core/entity/database-entity.decorator';
 
 @DatabaseEntity('Aser')
 export class Aser extends Entity {
-
-  static ReadingLevels = [
-    'Nothing',
-    'Read Letters',
-    'Read Words',
-    'Read Sentence',
-    'Read Paragraph',
-  ];
-  static MathLevels = [
-    'Nothing',
-    'Numbers 1-9',
-    'Numbers 10-99',
-    'Subtraction',
-    'Division',
-  ];
+  static ReadingLevels = ['Nothing', 'Read Letters', 'Read Words', 'Read Sentence', 'Read Paragraph'];
```

Comment:
is there a way to force consistent linebreaks (all or nothing) but leave the choice between breaks or not to the user? (eslint seems to have that option)

Reply:
apparently not possible with prettier ...